src/openrct2-ui/input/KeyboardShortcuts.h

```diff
@@ -46,6 +46,7 @@ enum
     SHORTCUT_SEE_THROUGH_SCENERY_TOGGLE,
     SHORTCUT_INVISIBLE_SUPPORTS_TOGGLE,
     SHORTCUT_INVISIBLE_PEOPLE_TOGGLE,
+    SHORTCUT_GRIDLINES_DISPLAY_TOGGLE,
```

Comment:
All new shortcuts must be added to the end of the list, otherwise users existing shortcuts will become malformed.